test/e2e/events_test.go

```diff
@@ -199,4 +199,19 @@ var _ = Describe("Podman events", func() {
 
 		wg.Wait()
 	})
+
+	It("podman events pod creation", func() {
+		create := podmanTest.Podman([]string{"pod", "create", "--infra=false", "--name", "foobarpod"})
+		create.WaitWithDefaultTimeout()
+		Expect(create).Should(Exit(0))
+		inspect := podmanTest.Podman([]string{"pod", "inspect", "foobarpod"})
+		inspect.WaitWithDefaultTimeout()
+		data := inspect.InspectPodToJSON()
```

Comment:
This is not necessary, you could get the id from the pod create command output.

Reply:
fixed